enhancements/observability/multicluster-observability-addon.md

```diff
@@ -0,0 +1,647 @@
+```
+
+Notice that secrets referenced on CLF can either exist in the managed cluster namespace or on the CLF namespace (i.e. `open-cluster-management-observability`) as is the case for `application-logs` secret:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: application-logs-loki-mtls-credentials
+  namespace: open-cluster-management-observability
+stringData:
+  'tls.crt': "TLS client certificate"
+  'tls.key': "TLS key"
+  'ca-bundle.crt': "Certificate Authority certificate"
+```
+
+In turn the addon will compile a `ManifestWork` for the managed cluster `spoke` as follows and pass it over it's WorkAgentController:
+
+```yaml
+kind: ManifestWork
+metadata:
+  name: addon-multicluster-observability-addon-deploy-0
+  namespace: spoke
+spec:
+  workload:
+    manifests:
+    - apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        name: cluster-logging
+        namespace: openshift-logging
+      spec:
+        channel: stable-5.9 # Pulled from the AddOnDeploymentConfig
+        installPlanApproval: Automatic
+        name: cluster-logging
+        source: redhat-operators
+        sourceNamespace: openshift-marketplace
+        startingCSV: cluster-logging.v5.9.0
+    - apiVersion: v1
+      kind: Secret
+      metadata:
+        name: cluster-logs-aws-cloudwatch-credentials
+        namespace: openshift-logging
+      data:
+        access_key_id: "Base64 access key id"
+        access_key_secret: "Base64 access key secret"
+    - apiVersion: v1
+      kind: Secret
+      metadata:
+        name: application-logs-loki-mtls-credentials
+        namespace: openshift-logging
+      data:
+        'tls.crt': "Base64 encoded TLS client certificate"
+        'tls.key': "Base64 endoded TLS key"
+        'ca-bundle.crt': "Base64 encoded Certificate Authority certificate"
+    - apiVersion: logging.openshift.io/v1
+      kind: ClusterLogForwarder
+      metadata:
+        name: mcoa-instance
+        namespace: openshift-logging
+      spec:
+        outputs:
+        - loki:
+            labelKeys:
+            - log_type
+            - kubernetes.namespace_name
+            - kubernetes.pod_name
+            - openshift.cluster_id
+          name: application-logs
+          type: loki
+          url: "https://loki-foo.grafana.com"
+          secret:
+            name: application-logs-loki-mtls-credentials
+        - name: cluster-logs
+          type: cloudwatch
+          cloudwatch:
+            groupBy: logType
+            region: eu-central-1
+          secret:
+            name: cluster-logs-aws-cloudwatch-credentials
+        pipelines:
+        - inputRefs:
+          - application
+          name: application-logs
+          outputRefs:
+          - application-logs
+        - inputRefs:
+          - audit
+          - infrastructure
+          name: cluster-logs
+          outputRefs:
+          - cluster-logs
+```
+
+#### Multi Cluster OTLP Collection and Forwarding
```

Comment:
@iblancasa could you please add a commit to this PR updating the resources for the tracing side? I'm mainly concerned with an up-to-date example of the `OpenTelemetryCollector` and the resulting resource in the `ManifestWorks`